src/pages/AddServers/PrepareAddServers.js

```diff
@@ -63,10 +65,10 @@ class PrepareAddServers extends BaseUpdateWizardPage {
       playbooks: ['commit']
```

Comment:
@dotNomad will update

Reply:
@dotNomad  
1. The API call need just name without '.yml' while step needs '.yml' to apply update during progress. 
2. If the name is used in multiple files...then they belong to common constants. Some action names are local I will just keep them in local. For playbooks...I move them to constants since it is too much to identify if they are used in multiple places or not.